docs/_config.yml

```diff
@@ -295,6 +295,9 @@ es:
       sections:
         - path: es/week11/11-1.md
         - path: es/week11/11-2.md
+    - path: es/week12/12.md
```

Comment:
My PR that's almost ready to get merged includes this, if you're concerned: https://github.com/Atcold/pytorch-Deep-Learning/pull/555/files

Reply:
OK so I will add mine using the same header, fine? 
i.e. this will be present
```
-path: es/week12/12.md
  sections:
    - path: es/week12/12-2.md
```

Reply:
I think it shouldn't create a merge conflict because it's exactly the same, but even if it does resolving it should be trivial.

Reply:
And how should it be resolved then? 

(A) by me **adding** yours  `- path: es/week12/12-3.md`
(B) by me **removing** all except for `- path: es/week12/12-2.md`

Am i being clear with my question? i.e. what do you want me to do about it?

Reply:
Typically, when your PR happens to be merged first, option (A) would apply. If my PR were to be merged first, then you'd just need to be sure that nothing got duplicated.

Reply:
Oh ok, will leave as is and keep an eye on what happens next. Thanks!